src/blog/2023/04/3-quick-node-red-tips-6.md

```diff
@@ -8,6 +8,7 @@ image: /images/blog/nr-quicktips.jpg
 tags:
     - posts 
```

Comment:
I've noticed the json files already have this tag in each folder, so there's no need to add this tag on each post
```suggestion
```